guides/release/community-release.md

```diff
@@ -0,0 +1,98 @@
+
+b. **Merge Freeze on the Main Branch:**
+
+- Notify your team about the merge freeze on the main branch.
+
+c. **Tagging Pull Requests:**
+
+- Add appropriate labels to all pull requests intended for the release to maintain a clean and organized changelog.
+
+### 3. Update Changelog
+
+a. **Pull Latest Changes:**
+
+- Navigate to the project directory and pull the latest changes on the main branch.
+
+b. **Change Log Branch**
+
+- Create a new branch for the release changelog.
+
+c. **Generate Changelog**
+
+- Utilize the [GitHub Changelog Generator CLI](https://github.com/github-changelog-generator/github-changelog-generator) to generate the changelog.
+
+CLI:
+
+```bash
+github_changelog_generator --since-tag= <<CURRENT_TAG>> --future-release= <<RELEASE_VERSION_TAG>> -t <<GITHUB_TOKEN>> -p <<PROJECT_NAME>> -u <<PROJECT_GROUP>> --base CHANGELOG.md
+```
+
+| Variables:          | Explanation:                                                               | Examples:                                                                                                                                 |
+| ------------------- | -------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| CURRENT_TAG         | Changelog will start after specified tag.                                  | 2.1.0                                                                                                                                     |
+| RELEASE_VERSION_TAG | Release version                                                            | 2.2.0                                                                                                                                     |
+| PROJECT_NAME        | Name of project on GitHub                                                  | agent, helm-charts, wanda or web                                                                                                          |
+| GITHUB_TOKEN        | Your personal Github Token                                                 | https://docs.github.com/en/enterprise-server@3.6/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens |
+| PROJECT_GROUP       | Username of the GitHub repository OR namespace of target Github repository | trento-project                                                                                                                            |
+
+CLI Example:
+
+```bash
+github_changelog_generator --since-tag=2.1.0 --future-release=2.2.0 -p agent -t <<GITHUB_TOKEN>> -u trento-project --base CHANGELOG.md
+```
+
+### 4. Version Bump
+
+a. **Version Upgrade**
+
+- Search for the current version in every project file and perform a version bump.
+
+Previous Release Examples:
+
+- [Agent](https://github.com/trento-project/agent/commit/df9bce2692ee46d3faa548494ec7ba40a22d1873)
```

Comment:
I still think that a link to a release would be better, as some release PRs might have multiple commits